server/utils/passkey/session.ts

```diff
@@ -0,0 +1,76 @@
+import { UserProfile } from '@prisma/client'
+import jwt from 'jsonwebtoken'
+
+const jwtSecret = process.env.SUPABASE_JWT_SECRET || ''
+const jwtIssuer = process.env.SUPABASE_URL || ''
+
+export function createWebAuthnAccessTokenForUser(user: UserProfile) {
+  const issuedAt = Math.floor(Date.now() / 1000)
+  const expirationTime = issuedAt + 3600 // 1 hour expiry
+  
+  // Create a payload that matches Supabase's expected format
+  const payload = {
+    iss: jwtIssuer,
+    sub: user.supId,
+    aud: 'authenticated',
+    exp: expirationTime,
+    iat: issuedAt,
+    
+    email: user.supEmail,
+    phone: user.supPhone,
+    app_metadata: {}, // Add app_metadata field
```

Comment:
We set some custom metadata in `prisma/migrations/20250224085617_custom_access_token/migration.sql`, like `deviceNonce`, that we might need to have here.